index.js

```diff
@@ -42,16 +42,16 @@ function getAbi (target, runtime) {
 }
 
 var allTargets = [
-  {runtime: 'node', target: '0.10.48'},
-  {runtime: 'node', target: '0.12.17'},
-  {runtime: 'node', target: '4.6.1'},
-  {runtime: 'node', target: '5.12.0'},
-  {runtime: 'node', target: '6.9.1'},
-  {runtime: 'node', target: '7.0.0'},
-  {runtime: 'electron', target: '1.0.2'},
-  {runtime: 'electron', target: '1.2.8'},
-  {runtime: 'electron', target: '1.3.12'},
-  {runtime: 'electron', target: '1.4.10'}
+  {runtime: 'node', target: '0.10.48', lts: false},
+  {runtime: 'node', target: '0.12.17', lts: false},
+  {runtime: 'node', target: '4.6.1', lts: false},
```

Comment:
Oh I forgot Node 4.

Both Node 4 and Node 6 are active LTS releases at the moment.